app/src/main/java/ip/team13/petowner/ui/main/MainActivity.kt

```diff
@@ -15,21 +14,18 @@ class MainActivity : FragmentActivity() {
 
     private val viewModel: MainViewModel = MainViewModel()
 
-    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
-        super.onActivityResult(requestCode, resultCode, data)
```

Comment:
Does GoogleLoginFragment's onActivityResult(..) still work after removing it from activity?

Reply:
Yes, still working. The `onActivityResult` in `MainActivity` was empty, I don't see how it could influence anything🤔